packages/platform/examples/base/README.md

```diff
@@ -1,35 +1,16 @@
 # Atoms examples
 Example apps using atoms - customizable UI components to integrate scheduling into your product.
 
-## Running examples apps locally
-
-1. Cal's backend is required to run example apps, so clone `https://github.com/calcom/cal.com` and follow setup instructions in the readme. Importantly, you need to have Google credentials setup by following [this section](https://github.com/calcom/cal.com?tab=readme-ov-file#obtaining-the-google-api-credentials) in the docs.
+## Set up your sandbox
 
-2. Open "apps/api/v2/.env" and copy environment variables below, and then copy `NEXTAUTH_SECRET` from the root ".env" of repository cloned in step 1.
-```jsx
-NODE_ENV="development"
-API_PORT=5555
-DATABASE_READ_URL="postgresql://postgres:@localhost:5450/calendso"
-DATABASE_WRITE_URL="postgresql://postgres:@localhost:5450/calendso"
-API_URL="http://localhost"
-NEXTAUTH_SECRET="copy from .env of root of repository cloned in step 1"
-DATABASE_URL="postgresql://postgres:@localhost:5450/calendso"
-JWT_SECRET="asjdijI1JIO12I3O89198jojioSAJDU"
-REDIS_URL="redis://localhost:6379"
-```
+1. Open your browser at "https://app.cal.dev" and login with admin username `admin@example.com` and password `ADMINadmin2022!`.
+2. In the web app navigate to `https://app.cal.dev/settings/platform/new` and create a platform. When asked for phone verification code enter `111111`.
```

Comment:
this link doesnt exist yet but we should change the route and remove organization (the concept is different)